### PR TITLE
RIA-3313: Case Officer notification on FTPA submitted home office notification failure

### PR DESCRIPTION
**Pull request by jkumar-moj**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3313

### Change description ###
Case Officer notification on FTPA submitted home office notification failure

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/functionalTest/resources/scenarios/RIA-3313-appellant-ftpa-submitted-ho-notification-failure.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    "input": {
+      "id": 3313,
+      "eventId": "applyForFTPAAppellant",
+      "state": "ftpaSubmitted",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "ariaListingReference": "LP/12345/2019",
+          "listCaseHearingCentre": "bradford",
+          "homeOfficeFtpaAppellantInstructStatus": "FAIL"
```
Comment (Lee365) [inline src/functionalTest/resources/scenarios/RIA-3313-appellant-ftpa-submitted-ho-notification-failure.json:15]: Do these test need Launch Darkly check ?

Comment (jkumar-moj) [reply src/functionalTest/resources/scenarios/RIA-3313-appellant-ftpa-submitted-ho-notification-failure.json:15]: The changes are independent and can be run with or without the flag.

**Review by Lee365 — APPROVED** [review]
LGTM

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Review by Lee365 — APPROVED** [review]
LGTM

**Pull request merged**